src/app/domain/geolocation/tracking/index.js

```diff
@@ -120,6 +121,11 @@ export const setTrackingConfig = async newTrackingConfig => {
   await storeData(StorageKeys.GeolocationTrackingConfig, newTrackingConfig)
 }
 
+export const handleActivityChange = async event => {
+  Log('[ACTIVITY CHANGE] - ' + JSON.stringify(event))
```

Comment:
These log is duplicated with the one from the headless task no ?

`Log('[ACTIVITYCHANGE] -' + JSON.stringify(params))`

Reply:
Good catch :+1: